bundles/binding/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/HueGenericBindingProvider.java

```diff
@@ -84,10 +84,12 @@ public void processBindingConfiguration(String context, Item item,
 
 		super.processBindingConfiguration(context, item, bindingConfig);
 
-		try {
-
-			if (bindingConfig != null) {
+		if( bindingConfig == null || bindingConfig.length() < 1 ) {
+				logger.warn("No settings configured for Hue binding for item '{}'.", item);
```

Comment:
```java
		if (bindingConfig == null || bindingConfig.length() < 1)
```

I enthusiastically encourage you to install the IDE, if for no other reason than your valuable contributions will be so much easier to merge.  I appreciate the desire to maintain Luddite status — just get it done! :smile: — but in order to keep the code consistent for future changes, it really has to follow the formatting and cleanup standards enforced by the IDE.  Thanks again for contributing, Chris.